config.yaml

```diff
@@ -50,9 +50,26 @@ contact: 'hello@catalyst.coop'
 
 # Order of episodes in your lesson
 episodes: 
-- introduction.md
+# chapter 1. ingestion
+- introduction.md # hmm.
 - working-with-diverse-filetypes.md
 - downloading-files-programmatically.md
+- scraping data
+# chapter 2. visualization/reusability
+- plotting # pick a few transforms and do them - dropping nulls, adding dtypes, stacking, unit conversion
+- validation testing / properties of your data
+- abstraction/modularization
+- automated testing
+# chapter 3. when things go wrong
+- debuggers
+- profiling
```

Comment:
This _could_ get cut if we're tight on time.

Reply:
In fact, let's cut this 'cause it'll cause more setup, installation and explanation problems for us.